src/pages/event-attendees/event-attendees.ts

```diff
@@ -25,12 +25,8 @@ export class EventAttendeesPage {
   private qrRe: RegExp;
   private attendeesStorageKey: string;
 
-  constructor(private attendeesService: AttendeesService,
-              private storage: Storage,
-              private toastCtrl: ToastController,
-              private queueService: QueueService,
-              private networkCheckService: NetworkCheck) {
-
+  constructor(private attendeesService: AttendeesService, private storage: Storage,
+              private toastCtrl: ToastController, private queueService: QueueService, private networkCheckService: NetworkCheck, private alertCtrl: AlertController) {
```

Comment:
Here a small Lint issue too, maximum line length recommended under TS standards is 120.